backend/src/main/java/com/woowacourse/gongseek/comment/presentation/dto/CommentRequest.java

```diff
@@ -4,11 +4,13 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
 
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 @Getter
 public class CommentRequest {
 
+    @Length(max = 10000)
```

Comment:
```suggestion
    @Length(max = 10_000)
```
요렇게하면 더 가독성있을 것 같아요!